src/cephadm/cephadm

```diff
@@ -1755,8 +1755,8 @@ def get_container(fsid, daemon_type, daemon_id,
     ceph_args = [] # type: List[str]
     if daemon_type in Monitoring.components:
         uid, gid = extract_uid_gid_monitoring(daemon_type)
-        m = Monitoring.components[daemon_type]  # type: ignore
-        metadata = m.get('image', dict())  # type: ignore
+        # m = Monitoring.components[daemon_type]  # type: ignore
+        # metadata = m.get('image', dict())  # type: ignore
```

Comment:
can you remove this code altogether?

the commit message is a bit misleading. The code is used, but the `image` attribute is gone?

Reply:
forget that. `metadata` is unused.

Reply:
Then I suggest to not only remove the code I commented out, but also the other code as well that previously only has been commented out but belongs to the code I commented out. Would that be okay?